examples/mp-metrics-timed/src/test/java/WeatherServiceTest.java

```diff
@@ -0,0 +1,151 @@
+        assertTrue(metric.contains("application:weather_day_status_seconds_count 1.0"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_rate_per_second timer"));
+        assertTrue(metric.contains("application:weather_day_status_rate_per_second"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_one_min_rate_per_second timer"));
+        assertTrue(metric.contains("application:weather_day_status_one_min_rate_per_second"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_five_min_rate_per_second timer"));
+        assertTrue(metric.contains("application:weather_day_status_five_min_rate_per_second"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_fifteen_min_rate_per_second time"));
+        assertTrue(metric.contains("application:weather_day_status_fifteen_min_rate_per_second"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_min_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_min_seconds"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_max_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_max_seconds"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_mean_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_mean_seconds"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_stddev_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_stddev_seconds"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_seconds{quantile=\"0.5\"}"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_seconds{quantile=\"0.75\"}"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_seconds{quantile=\"0.95\"}"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_seconds{quantile=\"0.98\"}"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_seconds{quantile=\"0.99\"}"));
+        assertTrue(metric.contains("# TYPE application:weather_day_status_seconds timer"));
+        assertTrue(metric.contains("application:weather_day_status_seconds{quantile=\"0.999\"}"));
+    }
+
+    private void assertJsonFormat(final String metricPath) {
+        final String metric = WebClient.create(base.toExternalForm())
```

Comment:
maybe load it as JsonObject and assert keys(), will make it inline and more reasable

Reply:
Done